aws_pricing/README.md

```diff
@@ -0,0 +1,55 @@
+# Agent Check: AWS Pricing
+
+## Overview
+
+This check pulls in pricing information [published by AWS][1] to make it easier to measure cost of resource utilization in within Datadog.
+
+<div>Icons made by <a href="https://www.flaticon.com/authors/eucalyp" title="Eucalyp">Eucalyp</a> from <a href="https://www.flaticon.com/" 			    title="Flaticon">www.flaticon.com</a> is licensed by <a href="http://creativecommons.org/licenses/by/3.0/" 			    title="Creative Commons BY 3.0" target="_blank">CC 3.0 BY</a></div>
+
+## Setup
+
+### Installation
+
+The AWS Pricing check is not included in the [Datadog Agent][2] package, so you will
```

Comment:
Could you please inline [the official instructions to install an integrations from integrations-extras](https://docs.datadoghq.com/agent/guide/community-integrations-installation-with-docker-agent/) ?  Or at least link to it :)